src/mono/sample/iOS-NativeAOT/Program.csproj

```diff
@@ -9,7 +9,6 @@
     <RuntimeIdentifier>$(TargetOS)-$(TargetArchitecture)</RuntimeIdentifier>
     <AppName>HelloiOS</AppName>
     <UseNativeAOTRuntime Condition="'$(UseNativeAOTRuntime)' == ''">true</UseNativeAOTRuntime>
-    <PublishDir Condition="'$(PublishDir)' == ''">$(OutputPath)/publish</PublishDir>
```

Comment:
When building a sample locally, how does the output dir structure look like now?

Reply:
We match it with the AppleBuild.targets and the Mono sample. It should be `/src/mono/sample/iOS-NativeAOT/bin/ios-arm64/Bundle/HelloiOS` for ios-arm64.

Reply:
Thanks for clarification.